src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier randColor = new ColorSupplier();
+        Random rand = new Random();
```

Comment:
ColorSupplier and Random are also your dependencies here. Also, pay attention to your instances namings